cantools/database/can/c_source.py

```diff
@@ -41,6 +41,7 @@
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <stdlib.h>
```

Comment:
I think we should include stddef.h instead.

https://en.wikipedia.org/wiki/C_data_types#stddef.h

Reply:
Sounds good to me, changed!